client/src/components/GraduateForm.js

```diff
@@ -1,29 +1,47 @@
-import React, { useContext } from 'react';
+import React, { useContext, useState } from 'react';
 import { Formik, Form } from 'formik';
 import * as Yup from "yup";
 import FormField from "../constant/FormField";
 import StyledButton from '../constant/StyledButton';
 import { ProfileContext } from '../context/ProfileContext';
-
+import { skills } from '../api/skills';
 import styled from 'styled-components';
 
 
 const GraduateForm = () => {
+	const [newSkills, setNewSkills] = useState([]);
 	const { profiles, addProfile, profile, editProfile }  = useContext(ProfileContext);
-	console.log('add', profile);
 
 	const handleSubmit = (values) => {
-		const { name, email, website, phone } = values;
+		const { firstName, lastName, skills } = values;
 		const newProfile ={
-			'full_name':name,
-			'email':email,
-			'website':website,
-			'phone':phone,
+			'first_name':firstName,
+			'last_name':lastName,
+			'skills':skills,
 		};
 		addProfile(newProfile);
 	};
 
-	const initialValue = profile ? { name:`${profile.name}`, email:`${profile.email}`, website:`${profile.website}`, phone:`${profile.phone}` }:{ name:'', email:'', website:'', phone:'' };
+	const deleteSkill = (e)=>{
+		e.preventDefault();
+		let remainedSkills = newSkills.filter((skill)=>skill!==e.target.value);
+		setNewSkills(remainedSkills);
+	};
+	const handleValidate =async (e,setFieldValue)=>{
+		e.persist();
+		const res = await skills();
+		const response= res.map((x)=>x.toUpperCase());
+		let event = e.key;
+		let word = e.target.value.trim().toUpperCase();
+		if(event==' '){
+			response.includes(word) && !newSkills.includes(word) && setNewSkills([...newSkills, word]);
```

Comment:
With `response.includes(word)` you're stopping the graduate to be able to enter a new skill if it's outside the set list of skills. So at the moment I can't enter let's say "SQL" because it's not in the initial list. I think there are two ways to handle this:
1)  If you want the graduate to be able to choose from a set list of skills then there shouldn't be a free field to enter skills but could use a list of skills with checkboxes or a dropdown or something like that. The risk with this is that you might include something in a set list and would have to keep it updated with additional skills/tech.
2) If you want this to be a free field then you need to accommodate completely new skills to be entered (and just remove `response.includes(word)`. Just need to have a think about how the backend should handle skills as well to match this.